### PR TITLE
Improved formatting of SQL Database Type Providers walkthrough

### PR DESCRIPTION
**Pull request by svick**
- fixed code formatting in lists
- one-item lists are no longer lists
- removed `<br />`s
- fixed section link

This renders correctly for me in a local docfx build, but due to how fiddly formatting of code blocks in lists is, I can't guarantee that it will work the same on docs.MS.

cc: @cartermp

### CODE REVIEW

**Comment by qinezh** [comment]
Open Publishing Build Service: The pull request content has been published and here are some changed files links of this time:
- Docset 1 - **core-docs**  
  [https://review.docs.microsoft.com/en-us/dotnet/articles/fsharp/tutorials/type-providers/accessing-a-sql-database?branch=pr-en-us-1104](https://review.docs.microsoft.com/en-us/dotnet/articles/fsharp/tutorials/type-providers/accessing-a-sql-database?branch=pr-en-us-1104)  

**Status**: SucceededWithWarning. **Message**: Warning occurred: Unable to resolve cross-reference "<a href="xref:Microsoft.Win32.SafeHandles.SafeNCryptKeyHandle" sourcefile="docs/standard/garbagecollection/implementing-dispose.md" sourcestartlinenumber="413" sourceendlinenumber="413">SafeNCryptKeyHandle</a>", "<a href="xref:Microsoft.Win32.SafeHandles.SafeNCryptProviderHandle" sourcefile="docs/standard/garbagecollection/implementing-dispose.md" sourcestartlinenumber="413" sourceendlinenumber="413">SafeNCryptProviderHandle</a>", "<a href="xref:Microsoft.Win32.SafeHandles.SafeNCryptSecretHandle" sourcefile="docs/standard/garbagecollection/implementing-dispose.md" sourcestartlinenumber="413" sourceendlinenumber="413">SafeNCryptSecretHandle</a>"..  
[Open Publishing Report](https://opbuildstorageprod.blob.core.windows.net/report/2016%5C10%5C2%5C2a032abb-a76c-bfc8-4231-71ab612989a2%5CPullRequest%5C201610022010558443-1104%5Cworkflow_report.html).

File: docs/fsharp/tutorials/type-providers/accessing-a-sql-database.md [hunk 1]
```diff
@@ -428,7 +393,7 @@ CREATE TABLE [dbo].[Table1] (
   PRIMARY KEY CLUSTERED ([Id] ASC)
 );
 
---Create Table2.
+-- Create the Table3 table.
```
Comment (cartermp) [inline docs/fsharp/tutorials/type-providers/accessing-a-sql-database.md]: I think that this should be "Create the `Table2` table".

**Review by cartermp — APPROVED** [review]
LGTM modulo the single change mentioned.  Thanks, @svick!

**Comment by cartermp** [comment]
Oh, and it seems to render fine on staging as well.

File: docs/fsharp/tutorials/type-providers/accessing-a-sql-database.md (hunk 1 above)
Comment (svick) [inline docs/fsharp/tutorials/type-providers/accessing-a-sql-database.md]: Fixed. I have also added a comment about inserting to `Table3`.

**Comment by qinezh** [comment]
Open Publishing Build Service: The pull request content has been published and here are some changed files links of this time:
- Docset 1 - **core-docs**  
  [https://review.docs.microsoft.com/en-us/dotnet/articles/fsharp/tutorials/type-providers/accessing-a-sql-database?branch=pr-en-us-1104](https://review.docs.microsoft.com/en-us/dotnet/articles/fsharp/tutorials/type-providers/accessing-a-sql-database?branch=pr-en-us-1104)  

**Status**: SucceededWithWarning. **Message**: Warning occurred: Unable to resolve cross-reference "<a href="xref:Microsoft.Win32.SafeHandles.SafeNCryptKeyHandle" sourcefile="docs/standard/garbagecollection/implementing-dispose.md" sourcestartlinenumber="413" sourceendlinenumber="413">SafeNCryptKeyHandle</a>", "<a href="xref:Microsoft.Win32.SafeHandles.SafeNCryptProviderHandle" sourcefile="docs/standard/garbagecollection/implementing-dispose.md" sourcestartlinenumber="413" sourceendlinenumber="413">SafeNCryptProviderHandle</a>", "<a href="xref:Microsoft.Win32.SafeHandles.SafeNCryptSecretHandle" sourcefile="docs/standard/garbagecollection/implementing-dispose.md" sourcestartlinenumber="413" sourceendlinenumber="413">SafeNCryptSecretHandle</a>"..  
[Open Publishing Report](https://opbuildstorageprod.blob.core.windows.net/report/2016%5C10%5C2%5C2a032abb-a76c-bfc8-4231-71ab612989a2%5CPullRequest%5C201610022059580963-1104%5Cworkflow_report.html).

**Pull request merged**